src/sidebar/util/version-data.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+/**
+ * Build version data object for current session and client
+ *
+ * @param {Object} auth - Object representing auth state and user information
```

Comment:
It would be helpful for future readers to have more specific types here. Search for `@typedef` or `@prop` for examples of defining object types. IDEs can also take advantage of those types when writing code.